site/content/docs/main/guides/fips.md

```diff
@@ -101,7 +101,7 @@ Note that if you do build with FIPS mode outside of the build container, you can
 We can first compile the Envoy binary by running the following in a `bash` shell from the Envoy source directory:
 
 ```bash
-BAZEL_BUILD_EXTRA_OPTIONS="--define boringssl=fips" ENVOY_DOCKER_BUILD_DIR=<envoy-output-dir> ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release.server_only'
+BAZEL_BUILD_EXTRA_OPTIONS="--define boringssl=fips" ENVOY_DOCKER_BUILD_DIR=<envoy-output-dir> ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release //test/exe:envoy_static_test'
```

Comment:
Just a question: what are these [test targets](https://github.com/envoyproxy/envoy/blob/main/test/exe/BUILD)? I've used `//source/exe:envoy-static` instead (though I never dealt with FIPS builds).

Reply:
that particular test compiles the envoy binary target and then checks it is statically linked, the `bazel.release` "subcommand" of that script takes a test target to run so i chose that one as a minimal test that wouldn't add too much overhead to the build (bazel should cache the test compilation of the envoy binary and reuse it in the final binary build step since it is using all the same parameters)

Reply:
I decided to omit commenting on these details from this change but can add it if we think its useful

Reply:
Thanks for the info! Maybe it could be good to add some short version, since for someone who does not know (like me!) it might look a bit like a debug build due to the target name 😄